qt/python/mantidqt/gui_helper.py

```diff
@@ -83,24 +115,40 @@ def closeEvent(self, event):
         event.accept()
     '''
     try:
+        # try using built-in help in mantid
         import mantidqt
-        mantidqt.interfacemanager.InterfaceManager().showCustomInterfaceHelp(mantidplot_name)
+        mantidqt.interfacemanager.InterfaceManager().showCustomInterfaceHelp(mantidplot_name, section)
```

Comment:
This line will always fail. Try
`mantidqt.interfacemanager.InterfaceManager().showCustomInterfaceHelp('/'.join([mantidplot_name, section]))` instead, or you need to pass the parameters down the chain